src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,27 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = new Injector("manufacturers");
```

Comment:
@EliosPride dont create Injector instance, use it static methods in place where you need